src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -539,7 +539,7 @@ private void leaveTheRoom()
      *
      * @param chatRoomMember the new member that has just joined the room.
      */
-    protected void onMemberJoined(final ChatRoomMember chatRoomMember)
+    private void onMemberJoined(final ChatRoomMember chatRoomMember)
```

Comment:
Should this `chatRoomMember` be annotated `@NotNull` now too?

Reply:
Sure. I removed the `final`, since we don't usually use it and I don't see a special reason to have it.